cmd/tailscale/cli/set.go

```diff
@@ -43,11 +43,13 @@ type setArgsT struct {
 	advertiseDefaultRoute  bool
 	opUser                 string
 	acceptedRisks          string
+	profileName            string
 }
 
 func newSetFlagSet(goos string, setArgs *setArgsT) *flag.FlagSet {
 	setf := newFlagSet("set")
 
+	setf.StringVar(&setArgs.profileName, "name", "", "short name for the login profile")
```

Comment:
Feels like you'd want this to be `profile-name` or something more specific. Otherwise it gets confusing given that `hostname` is also settable (at least with `up`).

Reply:
Done.
However, I also removed `--profile-name` from `tailscale up`. I think we are at the point where we should not add more flags to `tailscale up` and start steering folks to `tailscale login` and `tailscale set`.

Reply:
Given it's `hostname` (one word) vs `profile-name` (two words), suggest either `profilename` or something like `nickname`.

Reply:
well `hostname` is a single word https://en.wikipedia.org/wiki/Hostname 🤪 

nickname sgtm